docs/content/ingestion/faq.md

```diff
@@ -2,6 +2,10 @@
 layout: doc_page
 ---
 
+## My Data isn't being loaded
+
+If you are trying to stream in historical (not current time) data into Druid and you are using the [serverTime](../ingestion/realtime-ingestion.html) rejection policy in your ingestion spec (the default rejection policy), Druid will not ingest this data as it is outside of the acceptable window period. You can verify this is what is happening by looking at the logs of your real-time process for log lines containing "ingest/events/*". These metrics will indicate the events ingested, rejected, etc. We recommend using batch ingestion methods for historical data or using the [messageTime](../ingestion/realtime-ingestion.html) rejection policy.
```

Comment:
IMO we should recommend using batch indexing or using realtime + serverTime in production. I think realtime + messageTime or even realtime + none is interesting for testing but I wouldn't want to recommend either for production.

It's also good to point out that batch indexing has its own time based dropping- if your data is outside of the intervals in the spec, it'll be dropped. Sometimes people run into that.